packages/modules/web-viewer-react/package.json

```diff
@@ -2,22 +2,28 @@
   "name": "@itwin/web-viewer-react",
   "version": "5.0.0-dev.7",
   "description": "Configurable iTwin.js viewer for web applications",
-  "main": "lib/cjs/index.js",
+  "type": "module",
   "module": "lib/esm/index.js",
```

Comment:
Same as in other package. `"module": "lib/esm/index.js"` can be removed and `lib/esm` changed to `lib`.